vault/resource_okta_auth_backend.go

```diff
@@ -204,6 +212,14 @@ func oktaAuthBackendResource() *schema.Resource {
 	}
 }
 
+func parseTTLForState(i interface{}) string {
+	d, err := parseutil.ParseDurationSecond(i)
+	if err != nil {
+		return i.(string)
```

Comment:
Is this definitely the right behaviour? I'm always very wary about throwing away `err` values.

Reply:
Yeah, I agree with you, but unfortunately `SchemaStateFunc` signature does not include returning `error`.  I suppose we could at least log the error, not sure if a panic is warranted here.

ref signature:
```
type SchemaStateFunc func(interface{}) string
```

Reply:
I think adding a `ValidateFunc` should mitigate this issue. Changes forthcoming.

Reply:
addressed in [641159f](https://github.com/hashicorp/terraform-provider-vault/pull/1123/commits/641159fbcad86ace9bcca6d7d0816a20f46a09fe)